include/mainwindow.h

```diff
@@ -195,12 +204,11 @@ private slots:
   void saveAsTriggered(char no);                           // MIA
   void revertTriggered(char no);                           // MIA
 
-
   void bootOptionTriggered();    //
   void toggleMiniModeTriggered();//
   void toggleShadeTriggered();   //
   void showLogWindowTriggered(); //
-
+  void diskBrowserTriggered();   // what are all these trailing comment markers for?
```

Comment:
I don't know. Perhaps a place holder for adding functionality comments in far future.